features/bevragen/protocollering/protocollering.feature

```diff
@@ -229,22 +230,18 @@ Functionaliteit: Als burger wil ik zien wie welke gegegevens van mij heeft gezie
       | <gevraagde rubrieken>       |
 
       Voorbeelden:
-      | fields                               | gevraagde rubrieken                                                                    |
-      | leeftijd                             | 010310                                                                                 |
-      | naam.voorletters                     | 010210                                                                                 |
-      | naam.volledigeNaam                   | 010210, 010220, 010230, 010240, 010410                                                 |
-      | adressering.aanschrijfwijze          | 010210, 010220, 010230, 010240, 010410, 016110, 050220, 050230, 050240, 050610, 050710 |
-      | adressering.adresregel1              | 081110, 081120, 081130, 081140, 081150, 081210, 081310, 081330                         |
-      | adressering.adresregel2              | 080910, 081110, 081120, 081160, 081170, 081210, 081310, 081340                         |
-      | partners.aangaanHuwelijkPartnerschap | 050610, 050620, 050630                                                                 |
+      | fields             | gevraagde rubrieken                    |
+      | leeftijd           | 010310                                 |
+      | naam.voorletters   | 010210                                 |
+      | naam.volledigeNaam | 010210, 010220, 010230, 010240, 010410 |
 
   Rule: Wanneer een veld wordt gevraagd waarbij meerdere velden worden meegeleverd voor het bepalen van het polymorfe type, dan worden alle daarvoor nodige velden in 'request_gevraagde_rubrieken' opgenomen.
 
     Abstract Scenario: Vragen om <fields> protocolleert ook benodigde velden voor het type
       Gegeven de persoon met burgerservicenummer '000000024' heeft de volgende gegevens
       | naam            | waarde     |
       | anummer (01.10) | 5264846450 |
-      Als gba personen wordt gezocht met de volgende parameters
+      Als personen wordt gezocht met de volgende parameters
```

Comment:
gevraagde rubrieken van nationaliteiten.nationaliteit en nationaliteiten.redenOpname verschillen met wat in [features/bevragen/persoon/nationaliteit/protocollering/protocollering-gevraagde-rubrieken.feature](https://github.com/BRP-API/Haal-Centraal-BRP-bevragen/pull/1758/files#diff-8f8bbd647539f53f16a2f2497b86ab4c5abf3bb98bf23a5375126aeccf1310a0) regel 26 en 29 staat. Welke moet worden gehanteerd?